.github/workflows/ci.yml

```diff
@@ -81,9 +81,7 @@ jobs:
         node-version: 10.x
 
     - name: Set up protoc
-      # TODO: switch back to arduino/setup-protoc after the pagination fix lands:
-      # https://github.com/arduino/setup-protoc/pull/9
-      uses: solo-io/setup-protoc@1f5d8b5
+      uses: arduino/setup-protoc@64c0c85
```

Comment:
I think specifying a version number is just as safe, no? And much more clear what the capabilities are (just go straight to the releases page)

Reply:
> I think specifying a version number is just as safe, no?

they're just git tags, so they can be moved and changed at any time, unfortunately.